### ISSUE TITLE
Recommendation list

### ISSUE THREAD

**Issue by patcharees** [HaojiHu/TIFUKNN#7]
How to make a recommendation list for each customer?

**Comment by HaojiHu** [comment]
Not sure about your question. We just select the k entries with the highest values in the each user' result vector as the recommendation list.

**Comment by VaibhavKSingh** [comment]
There are two csv files for each data, history and future. How do you generate recommendation for the future data. What I understand from the code is that you have divided the customers into training, testing and validation sets and the final recommendations are generated for the test set customers only. How do you reterive the top k elements for each customer?

**Comment by HaojiHu** [comment]
If you want to do for each customer, you just need to decide which time step is the next step to predict and generate user vector basd on baskets before the next time step. Then, the top k elements are the top k entries in the user vector for each customer.

**Comment by HaojiHu** [comment]
Besides, I think you may also want to consider the leave-one-out method (if you want to consider all the users), in which you generate each target user vector by considering all other users in the kNN search. In my setting, I only search kNN for each test user in the training user set.

**Comment by VaibhavKSingh** [comment]
Does the next_time_step model the number of next baskets to be predicted ? Like if we set it to be 2, then would it mean that we are predicting for next 2 baskets instead of one?

**Comment by HaojiHu** [comment]
No, I don't have this implementation here.
